docs/source/metrics.rst

```diff
@@ -105,6 +104,13 @@ This metrics API is independent of PyTorch Lightning. Metrics can directly be us
     # total accuracy over all validation batches
     total_valid_accuracy = valid_accuracy.compute()
 
+.. warning::
```

Comment:
Can we rather make this a note than a warning? I think `warning` would mean that there are some issues on our side.

Reply:
You are right, have changed it 👍